pkg/build/node/node.go

```diff
@@ -460,7 +460,7 @@ func (c *BuildContext) prePullImages(dir, containerID string) error {
 				fmt.Printf("Pulling: %s\n", image)
 				err := docker.Pull(image, 2)
 				if err != nil {
-					return err
+					log.Warnf("Failed to pull %s with error: %v", image, err)
```

Comment:
if we are not able to pull one of the required images we don´t fail?
 will not fail later trying to setup the cluster?